tests/lit-tests/ind-var-simplify.ispc

```diff
@@ -0,0 +1,13 @@
+// RUN: %{ispc} %s --nostdlib --emit-llvm-text --target=avx2-i32x4 --arch=x86-64 -o %t.ll
+// RUN: FileCheck %s --input-file=%t.ll
+
```

Comment:
// REQUIRES: X86_ENABLED

Reply:
Actually this test should probably can use `--target=host`  right?

Reply:
The test needs to run for 64 bit arch, I think it's good enough to run for x86 only and for a one fixed target (not host).